src/fhirpath/search.py

```diff
@@ -354,7 +354,7 @@ def build_query_params_string(self, params):
         # Otherwise, an expection is raised for None values
         params = {k: v for k, v in params.items() if v is not None}
         return URL.build(
-            query={**params, **{"_type": self.context.resource_types}}
+            query={**params, **{"_type": ",".join(self.context.resource_types)}}
```

Comment:
did you mean to commit that in this PR ? doesn't it fix the arkhn/warehouse-api#169 issue ?

Reply:
Hum, I think it was related to the solving of this issue, yeah! I thought it was something I had forgotten from another PR
Do you think I should split it in 2 PRs (or at least 2 commits)?

Reply:
two commits and then rebase sounds good :)

Reply:
Done 🙂